decidim-surveys/app/views/decidim/surveys/surveys/show.html.erb

```diff
@@ -3,89 +3,101 @@
   description: translated_attribute(survey.description),
 }) %>
 
-<div class="row column view-header">
-  <h2 class="heading2"><%= translated_attribute survey.title %></h2>
-</div>
-
-<div class="row column">
+<div class="row columns">
+  <h2 class="section-heading"><%= translated_attribute survey.title %></h2>
   <div class="row">
-    <div class="columns medium-7 mediumlarge-8">
-      <div class="section">
-        <%== translated_attribute survey.description %>
-      </div>
+    <div class="columns large-6 medium-centered lead">
+      <%== translated_attribute survey.description %>
+    </div>
+  </div>
+</div>
 
-      <% unless survey.published? %>
-        <div class="section">
-          <div class="callout warning">
-            <h5><%= t('.survey_not_published.title') %></h5>
-            <p><%= t('.survey_not_published.body') %></p>
+<div class="row">
+  <div class="columns large-6 medium-centered">
+    <div class="card">
+      <div class="card__content">
+        <% unless survey.published? %>
+          <div class="section">
+            <div class="callout warning">
+              <h5><%= t('.survey_not_published.title') %></h5>
+              <p><%= t('.survey_not_published.body') %></p>
+            </div>
           </div>
-        </div>
-      <% else %>
-        <% if current_user %>
-          <% if survey.answered_by?(current_user) %>
-            <div class="section">
-              <div class="callout success">
-                <h5><%= t('.survey_answered.title') %></h5>
-                <p><%= t('.survey_answered.body') %></p>
+        <% else %>
```

Comment:
I hate these `unless ... else ...`, I feels it's hard to grasp the logic. Can you switch the order of the blocks?

```
if survey.published?
  <the part below>
else
  <the part above
```

You can do this in another PR